app.json

```diff
@@ -5,7 +5,7 @@
     "version": "1.0.1",
     "orientation": "portrait",
     "icon": "./assets/icon.png",
-    "scheme": "wavlake",
+    "scheme": ["nostr+walletconnect"],
```

Comment:
Tried appending the NWC scheme to the `wavlake` entry, but it threw a warning on not knowing which one to prioritize, will need to revisit if adding non-NWC deep links in the future.

Reply:
was the `wavlake` entry used for anything before? i couldn't track down any examples

Reply:
not that im aware of, i didnt find any either